packages/framework/build.xml

```diff
@@ -403,4 +401,717 @@
         </delete>
     </target>
 
+    <target name="checks" depends="db-check-mapping,standards,tests" description="Checks ORM mapping, coding standards and runs unit, DB and smoke tests."/>
```

Comment:
I would rather see these targets at the beginning of the file, along with the other "top-level" targets like `build`, `build-deploy-part-1-db-independent`, etc.

Reply:
I didn't get to reordering the file yet. I just appended the whole `build-dev.xml` so it's easy to validate  they have been simply merged together and nothing is missing.

Reply:
I plan to sort the targets alphabetically in a separate PR after this one gets merged.

Reply:
@henzigo I've added other missing checks into all `checks(-*)` targets. See hidden task `checks-internal`.